libs/cart/src/drivers/interfaces/cart-payment-methods-service.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { InjectionToken } from '@angular/core';
+import { DaffCartPaymentMethod } from '../../models/cart-payment';
+import { DaffCart } from '../../models/cart';
+
+/**
+ * The interface responsible for retrieving the available payment methods of a cart.
+ */
+export interface DaffCartPaymentMethodsServiceInterface<T extends DaffCartPaymentMethod> {
+	/**
+	 * List the available payment methods of a cart.
+	 */
+	list(cartId: DaffCart['id']): T[];
```

Comment:
This should probably return `Observable<T[]>`.

Reply:
Yes, fixed, nice catch.